kong/tracing/instrumentation.lua

```diff
@@ -149,6 +148,7 @@ function _M.http_client()
     end
 
     local span = tracer.start_span("HTTP " .. method .. " " .. uri, {
+      span_kind = 3,
```

Comment:
This attribute seems not really used in `start_span`.

Reply:
https://github.com/Kong/kong/blob/4d29046008daa56b717b69b94acfec76c635a7cf/kong/pdk/tracing.lua#L178
Thanks for porting it out, it should be `options.span_kind` there.